contracts/src/rollup/RollupUserLogic.sol

```diff
@@ -197,13 +197,15 @@ contract RollupUserLogic is RollupCore, UUPSNotUpgradeable, IRollupUser {
             "STAKED_ON_ANOTHER_BRANCH"
         );
 
-        uint256 timeSincePrev = block.number - getAssertionStorage(prevAssertion).createdAtBlock;
-        // Verify that assertion meets the minimum Delta time requirement
-        require(timeSincePrev >= minimumAssertionPeriod, "TIME_DELTA");
-
-        bytes32 newAssertionHash = createNewAssertion(assertion, prevAssertion, expectedAssertionHash);
+        (bytes32 newAssertionHash, bool overflowAssertion) = createNewAssertion(assertion, prevAssertion, expectedAssertionHash);
         _stakerMap[msg.sender].latestStakedAssertion = newAssertionHash;
 
+        if (!overflowAssertion) {
+            uint256 timeSincePrev = block.number - getAssertionStorage(prevAssertion).createdAtBlock;
+            // Verify that assertion meets the minimum Delta time requirement
+            require(timeSincePrev >= minimumAssertionPeriod, "TIME_DELTA");
+        }
```

Comment:
This would apply to the last of a batch of overflow assertions, where the 1st overflow assertion can be made immediately after the previous assertion but the last overflow assertion can only be made `minimumAssertionPeriod` after the `last-1` overflow assertion in the same batch. Overall its probably still fine, tho a little bit weird you have to wait `minimumAssertionPeriod` before making the last overflow assertion.

Reply:
@PlasmaPower what do you think? Should we change in a future PR?

Reply:
I think it's fine to do this as-is. The same number of assertions get a minimumAssertionPeriod bypass regardless. If you have a chain of assertions 1 through 5, and 2 through 4 are overflow assertions, in this PR after 1 gets created you can create 2 through 4 immediately, and then you have to wait before creating 5. If you looked at the last assertion, you'd need to wait before creating assertion 2, but then you could create assertions 3 through 5 immediately. That's conceptually a bit nicer I think, because assertions 3 through 5 are really continuations of assertion 2, but it adds complexity and I don't think it matters much in practice.